libs/application/templates/parental-leave/src/lib/parentalLeaveUtils.ts

```diff
@@ -320,12 +326,31 @@ export const calculatePeriodPercentage = (
   return Math.min(100, Math.round((months / difference) * 100))
 }
 
+const getOrFallback = (condition: Boolean, value: number | undefined) => {
+  if (condition === YES) {
+    // In the first version of the app, we can't manually change the number of
+    // days requested or given, so we use the maximum number of days in this case
+    if (value === undefined) {
+      return maxDaysToGiveOrReceive
+    }
+
```

Comment:
I believe if you initialise the value parameter with maxDaysToGiveOrReceive (`value = maxDaysToGiveOrReceive`) you can skip this.

Reply:
Yes! thanks